monitoring/docker/docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: "3.3"
+volumes:
+  grafana_data: {}
+
+services:
+
+  prometheus:
+    image: prom/prometheus
+    container_name: bor-prometheus
+    network_mode: host
+    volumes:
+      - ./prometheus.yml:/etc/prometheus/prometheus.yml
+    command:
+      - '--config.file=/etc/prometheus/prometheus.yml'
+      - '--storage.tsdb.path=/prometheus'
+      - '--web.console.libraries=/usr/share/prometheus/console_libraries'
+      - '--web.console.templates=/usr/share/prometheus/consoles'
+    ports:
+      - 9090:9090
```

Comment:
This will start at the same port as Heimdall Grafana. It would be great if we can use `9091:9090`. This will start Prometheus on `9091` port for Bor.

Reply:
```suggestion
      - 9091:9090
```